README.md

```diff
@@ -253,6 +254,17 @@ df.plot_bokeh.line(
 
 ![ApplevsGoogle_3](Documentation/Images/ApplevsGoogle_3.png)
 
+#### Lineplot with rangetool
+
+```python
+df = pd.DataFrame(np.random.randn(1000, 4), index=ts.index, columns=list('ABCD'))
+df = df.cumsum()
+
+df.plot_bokeh(rangetool=True)
+```
+
+![rangetool](Documentation/Images/rangetool.png)
```

Comment:
Would be nice if you could add a rangetool.gif animation, such that everyone can see how cool it works ;)

Reply:
Done!